topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial-ES.md

```diff
@@ -0,0 +1,809 @@
+>         - *"Replace with"*: `genotype`
+>
+>    Ahora queremos únicamente la columna que contiene la información de genotipo -  al final agregaremos esto en la anotación celular del objeto AnnData.
+>
+> 2. {% tool [Cut columns from a table](Cut1) %} con los siguientes parámetros:
+>    - *"Cut columns"*: `c9`
+>    - *"Delimited by"*: `Tab`
+>    - {% icon param-file %} *"From"*: la salida de la herramienta **Replace text** {% icon tool %}
+>
+> 3. Cambiar el nombre {% icon galaxy-pencil %} de la salida a `Genotype metadata`
+{: .hands_on}
+
+Es posible que desees hacer esto con distintos tipos de metadatos - qué laboratorios manejaron las muestras, en qué día se procesaron, etc. Una vez que hayas agregado todas tus columnas de metadatos, podemos entonces adherirlas unas a otras antes de conectarlas al objeto AnnData.
+
+> ### {% icon hands_on %} Práctica: Combinar las columnas de metadatos
+>
+> 1. {% tool [Paste two files side by side](Paste1) %} con los siguientes parámetros:
+>    - {% icon param-file %} *"Paste"*: `Genotype metadata`
+>    - {% icon param-file %} *"and"*: `Sex metadata`
+>    - *"Delimit by"*: `Tab`
+> 2. Cambiar el nombre {% icon galaxy-pencil %} de la salida a `Cell Metadata`
+{: .hands_on}
+
+¡Agrégalo al objeto AnnData!
+
+> ### {% icon hands_on %} Práctica: Añadir los metadatos al objeto AnnData
+>
+> 1. {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.7.5+galaxy0) %} con los siguientes parámetros:
+>    - {% icon param-file %} *"Annotated data matrix"*: la salida de la herramienta **Manipulate AnnData** {% icon tool %}
+>    - *"Function to manipulate the object"*: `Add new annotation(s) for observations or variables`
+>    - *"What to annotate?"*: `Observations (obs)``
+>    - {% icon param-file %} *"Table with new annotations"*: `Cell Metadata`
+{: .hands_on}
+
+¡Perfecto! Ahora puedes ejecutar **Inspect AnnData** para verificación, pero primero limpiemos un poco más este objeto. Sería mucho mejor si "lote" (`batch`) significara algo, en lugar de "el orden en el que la herramienta ‘Manipulate AnnData’ agregó mis conjuntos de datos".
+
+> ### {% icon hands_on %} Práctica:  Etiquetado de lotes
+>
+> 1. {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.7.5+galaxy0) %} con los siguientes parámetros:
+>    - {% icon param-file %} *"Annotated data matrix"*: la salida de la herramienta **Manipulate AnnData - Add new annotations** {% icon tool %}
+>    - *"Function to manipulate the object"*: `Rename categories of annotation`
+>    - *"Key for observations or variables annotation"*: `batch`
+>    - *"Comma-separated list of new categories"*: `N701,N702,N703,N704,N705,N706,N707`
+{: .hands_on}
+
+¡Ya casi llegamos a donde queremos! Sin embargo, mientras que nos hemos enfocado en los metadatos de nuestras células (muestra, lote, genotipo, etc.) para etiquetar las ‘observaciones’ en nuestro objeto...
```

Comment:
Ahh, no - it's because in that section we relabelled the cells (obs), and that leads into the next section relabelling the genes (vars)
A cunning link between sections!